camilo-alfonso/tarea-1/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="main">
+        <div class="main__header">
+            <img src="img/bbc_logo.png" alt="bbc logo" height="28px" />
+            <p class="main__header--title">WHY LIFE ON EARTH WASN'T AN ACCIDENT</p>
+        </div>
+        <div class="main__content">
+            <h1 class="main__content--title">FOCUS</h1>
+            <ul class="main__content--categories">
+                <li>SCIENCE</li>
+                <li>|</li>
+                <li>TECHNOLOGY</li>
+                <li>|</li>
+                <li>HEALTH</li> 
+            </ul>
+            <div class="main__cover">
+                <h1 class="main__cover--title">
+                A<br />
+                NEW <br/>RACE TO THE <span>MOON</span> HAS BEGUN
+                </h1>
+                <img src="img/moon_image.png" alt="moon image" />
+            </div>
+            <div class="main__paragraph">
+                <p class="main__paragraph--text">
+                    What China's pioneering lunar mission means<br />
+                    for the future of space exploration
+                    </p>
+                    <img class="main__paragraph--img" src="/img/bar_code.jpg" alt="bar code">
```

Comment:
Pleas use relative paths

```suggestion
                    <img class="main__paragraph--img" src="img/bar_code.jpg" alt="bar code">
```

Reply:
Hi. I fixed the image path issue.